Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19.4-alpine3.16 as builder
+FROM --platform=$BUILDPLATFORM golang:1.19.4-alpine3.16 as builder
```

Comment:
This is needed since we're doing cross compilation. Github Actions runners are AMD machines, so buildx builders are also AMD machines. We either have to cross-compile, supplying `GOOS=$TARGETOS GOARCH=$TARGETARCH` to `make all` or set up qemu emulation. Cross-compilation is much faster. You can see it [here in build log](https://github.com/kkopachev/kube-no-trouble/actions/runs/4250039056/jobs/7390760231#step:4:241) and few lines above [same for amd64](https://github.com/kkopachev/kube-no-trouble/actions/runs/4250039056/jobs/7390760231#step:4:235)

Reply:
Ah, ok, thanks for explaining this @kkopachev, I now better understand what are we trying to do here, i.e. the idea is to use the `platforms` parameter of the GH action to set only the target platform of the final image, but actually do a cross-compile (always within `amd64` platform/image). _(I agree that cross- is gonna be much faster 👍 )_